OasisDEX/oasis-borrow
# 🧭 Display human readable "time ago" in Discover

# [🧭 Display human readable "time ago" in Discover](https://app.shortcut.com/oazo-apps/story/6577/display-human-readable-time-ago)

![image](https://user-images.githubusercontent.com/16230404/197824718-6f435856-8922-4812-b1fb-e2746a33b5ae.png)
![image](https://user-images.githubusercontent.com/16230404/197824793-c6ad1bb2-a3ae-4cc9-8985-ee77810bf0cb.png)

## Changes 👷‍♀️

- Added `timeAgo` method to utils,
- updated Discover tables to display more flexible time with `timeAgo` instead of fixed "x days ago".
  
## How to test 🧪

See for "Highest Multiply P&L", "Most Yield Earned" or "Largest Debt" Discover pages and check out "activity column.
